index.rst

```diff
@@ -890,10 +800,15 @@ More details to follow.
 Telescope Cabinet Switchable PDU
 --------------------------------
 
-In the event that a controller in the cabinet needs power cycling remotely, this may be done by logging into the switchable PDU mounted in the cabinet. The IP and connection info can be found `here <https://confluence.lsstcorp.org/x/qw6SBg>`_
+In the event that a controller in the cabinet needs power cycling remotely,
+this may be done by logging into the switchable PDU mounted in the cabinet.
+The IP and connection info can be found `here <https://confluence.lsstcorp.org/x/qw6SBg>`_
 
-* Channel 1 is connected to the main 24V supply. This will power off the cRIO (and possibly the Copley controllers, Pilz Device, and Smart Relay).
-* Channel 2 is connected to powerbar in bottom of cabinet, which has the 220V connection to the mount (which powers the Embedded PC for the Collimation Camera) as well as the hexapod connected to it.
+* Channel 1 is connected to the main 24V supply. This will power off the cRIO
+(and possibly the Copley controllers, Pilz Device, and Smart Relay).
+* Channel 2 is connected to powerbar in bottom of cabinet, which has the 220V
```

Comment:
This is where the ion pump is now connected and everything on the spectrograph. Put a huge warning here saying not to touch it.